pkgs/applications/office/csv2odf/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, python3, fetchurl }:
+
+let
+  inherit (python3.pkgs) buildPythonApplication;
+in
+
+buildPythonApplication rec {
+  pname = "csv2odf";
+  version = "2.09";
+  src = fetchurl {
+    url = "mirror://sourceforge/project/${pname}/${pname}-${version}/${pname}-${version}.tar.gz";
+    sha256 = "09l0yfay89grjdzap2h11f0hcyn49np5zizg2yyp2aqgjs8ki57p";
+  };
+
+  meta = {
+    homepage = http://csv2odf.sourceforge.net;
```

permanent redirect to
```suggestion
    homepage = "https://sourceforge.net/p/csv2odf/wiki/Main_Page/";
```